src/QLContext.h

```diff
@@ -409,10 +409,13 @@ struct UnboundCollectionContext : ReferenceCounted<UnboundCollectionContext>, Fa
 	Reference<DirectorySubspace> metadataDirectory;
 
 	Reference<UnboundQueryContext> cx;
-	std::map<std::string, std::set<IndexInfo, index_compare>>
-	    simpleIndexMap; // This is used by the planner, and should only hold indexes that are ready to speed up queries
-	std::vector<IndexInfo> knownIndexes; // This holds all indexes that will be loaded as plugins (i.e. for sets and
-	                                     // clears), and should include indexes that are still building
+
+	// This is used by the planner, and should only hold indexes that are ready to speed up queries
+	std::map<std::string, std::set<IndexInfo, IndexComparator>> simpleIndexMap;
```

Comment:
Can we take the chance to rename this variable? It confused me when I first saw it as in I thought it only holds simple index, not the compound ones...

Reply:
I thought of changing it, but after looking at it more carefully, I realized it does only hold the simple indexes. Even for the compound indexes, it holds first part of the compound index, hence the name I guess.

Ideally, we should have just one map. That would be better cleanup.